designPatterns/modelViewController/what/text.md

```diff
@@ -20,16 +20,16 @@ Decouple data, presentation, and control logic of an application by separating t
 
 * _View_: Displays data, interacts with the user, and pulls data from the model if necessary.
 * _Controller_: Detects UI events such as mouse clicks, button pushes and takes follow up action. Updates/changes the model/view when necessary.
-* _Model_: Stores and maintains data. Updates views if necessary.
+* _Model_: Stores and maintains data. Updates view if necessary.
```

Comment:
I suggest changing "view" to "the view" for consistency with the other 2 points.